tests/schemas/input/test_json.py

```diff
@@ -11,7 +11,7 @@
 """
 
 root = pathlib.Path(__file__).parent.parent
```

Comment:
still failing at https://github.com/citation-style-language/schema/runs/725123572#step:5:53.

I think the issue is that `root` above is no longer correct. We need to add more parent directories like

```suggestion
root = pathlib.Path(__file__).parent.parent.parent.parent
```